docs/adr/0006-treat-rollouts-differently.md

```diff
@@ -0,0 +1,31 @@
+# Opting out of studies should not affect rollouts
+
+> Status: Open
+
+> Author: Travis Long
+
+> Deciders: Nimbus team, Trust & Legal
+
+> Date: H1 2022
+
+## Context and Problem Statement
+
+Rollouts are a feature of Nimbus and are distinct from experiments in that they are not measurement instruments but instead represent off-train configuration updates.
+
+Rollouts can serve many purposes:
+- Launching a winning branch of an experiment faster than the trains.
+- Launching a configuration to non-experiment users during an experiment after a short period of verification
+- Configuring different settings for a feature for different audiences remotely
+- A "kill switch" if you want to launch a feature but then turn it off if something goes wrong.
+
+Currently, rollouts are ignored if the users has opted out of studies because Nimbus treats rollouts the same as studies, obeying the user preference settings for experiments.
+
+## Decision Outcome
+
+Because of the differences between rollouts and experiments, the Nimbus team would like to treat them independently from each other.
+
+This would include:
```

Comment:
Does this also include relaxing the constraint that we can only support one rollout out per feature at a time?

Reply:
I hadn't considered that for this particular brief, I was only focusing on the subject of whether we treat rollouts the same in regards to the opt-out preference for studies. In my opinion, this makes sense to treat rollouts differently and not block the feature with them